newrelic-api-client/src/main/java/com/ocadotechnology/newrelic/apiclient/model/dashboards/dashboard/DashboardVersion.java

```diff
@@ -0,0 +1,15 @@
+package com.ocadotechnology.newrelic.apiclient.model.dashboards.dashboard;
```

Comment:
How is this clas used?

Reply:
It had to be a leftover from the previous commit. I have rebased changes and `DashboardVersion` is no longer in the changes.